### ISSUE TITLE
Implement Jest and Enzyme testing frameworks

### ISSUE THREAD

**Issue by dwalintukan** [bodhiproject/bodhi-ui#699]

**Comment by BernardHan** [comment]
(Do we need to test `stores/model/*` ?)
## Large Workload:
`stores/WalletStore.js`
`scenes/Event/store.js`
`scenes/CreateEvent/store.js`

## Medium Workload:
`stores/AllEventsStore.js`
`stores/BotCourtStore.js`
`stores/GlobalStore.js`
`stores/AppStore.js (do we need to test this?)`

`scenes/Wallet/History/store.js`
`scenes/QtumPrediction/store.js`
`scenes/Activities/Withdraw/store.js`
`scenes/Activities/ResultSetting/store.js`
`scenes/Activities/Finalize/store.js`
`scenes/Activities/ActivityHistory/store.js`

`components/PendingTxsSnackbar/store.js`

## Small Workload:
`stores/RefreshingStore.js`
`stores/UiStore.js`

`components/GlobalSnackbar/store.js`
`components/WalletUnlockDialog/store.js`

**Comment by dwalintukan** [comment]
@BernardHan 
Yes, models need to get tested too. Those tests are pretty critical to make sure the data gets set right and also to test the funcs that change the data inside those models. If you think about it, most data is derived from the models in the UI layer.

As for organization, let's do it this way. When you start on a store, create an issue then and place it in the `In Progress` column of the sprint board. That we you know which ones are being worked on at the moment.

You can also pre-assign the stores to test for each of you, but would also like to have an in-progress issue on the sprint board for tracking as well.
